docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  db:
+    image: postgres:11.2
```

Comment:
On checking the postgres running on live at the moment is only `9.4`, having this set higher locally could make things harder to debug.